CRISPResso2/CRISPRessoPooledCORE.py

```diff
@@ -647,12 +649,60 @@ def main():
 
             N_READS_ALIGNED=get_n_aligned_bam(bam_filename_amplicons)
 
-            s1=r"samtools view -F 4 %s 2>>%s | grep -v ^'@'" % (bam_filename_amplicons,log_filename)
-            s2=r'''|awk '{ gzip_filename=sprintf("gzip >> OUTPUTPATH%s.fastq.gz",$3);\
-            print "@"$1"\n"$10"\n+\n"$11  | gzip_filename;}' '''
+            if args.amplicons_file != '':
```

Comment:
My bad -- in this section of code, the amplicons_file will always be provided. 
Here, we can just separate reads out by chromosome name and ignore the position.
This complex case is line ~867 (with the comment "next, create the general demux command") when we need to keep track of the position as well as the chromosome. However, that part is much more hairy and will need a lot of work.